src/store/modules/consents.ts

```diff
@@ -245,9 +245,7 @@ export default vuexStoreModule({
           category: "preferences"
         });
         if (!consentGiven) {
-          ["jwt", "cookie:accepted", "consents"].forEach(k =>
-            localStorage.removeItem(k)
-          );
+          localStorage.clear();
```

Comment:
So with Ali's comment and your response in mind, do you plan to change this to explicitly remove items?

Reply:
Yup, I'll close this one and create a new PR which will have all the items for deletion specified.

Reply:
OK, go ahead!